common/bcp47/timezone.xml

```diff
@@ -19,12 +19,12 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="amevn" description="Yerevan, Armenia" alias="Asia/Yerevan"/>
             <type name="ancur" description="Curaçao" alias="America/Curacao"/>
             <type name="aolad" description="Luanda, Angola" alias="Africa/Luanda"/>
-            <type name="aqams" description="Amundsen-Scott Station, South Pole" deprecated="true" preferred="nzakl"/>
+            <type name="aqams" description="Amundsen-Scott Station, South Pole" deprecated="true" preferred="aqmcm"/>
             <type name="aqcas" description="Casey Station, Bailey Peninsula" alias="Antarctica/Casey"/>
             <type name="aqdav" description="Davis Station, Vestfold Hills" alias="Antarctica/Davis"/>
             <type name="aqddu" description="Dumont d'Urville Station, Terre Adélie" alias="Antarctica/DumontDUrville"/>
             <type name="aqmaw" description="Mawson Station, Holme Bay" alias="Antarctica/Mawson"/>
-            <type name="aqmcm" description="McMurdo Station, Ross Island" alias="Antarctica/McMurdo"/>
+            <type name="aqmcm" description="McMurdo Station, Ross Island" alias="Antarctica/McMurdo Antarctica/South_Pole"/>
```

Comment:
Should also have *removed* the Antarctica/South_Pole alias from Auckland, not doing that caused huge problems in ICU integration. I will make a separate PR to fix that.

Reply:
whoops